src/pages/careers/_who-we-look-for.js

```diff
@@ -80,22 +80,12 @@ const ImageWrapper = styled.div`
     max-width: 65rem;
 `
 const query = graphql`
-    query {
+    {
```

Comment:
[Clarification] Any reason this is deleted?

Reply:
Note: This was part of the codemod migration as query wrappers are no longer required. They can all be deleted in the future.